client/scss/components/_promo.scss

```diff
@@ -142,6 +142,10 @@
   left: 0;
   padding: map-get($gutter-width, 'small')/2;
 
+  .promo--on-cream & {
```

Comment:
Yea, but I think we've used `--cream` else where.

It also suggests that the component should know about it's surrounding, which, presentationally, I'd argue it shouldn't.

Otherwise it would be the cream container passing that value down... which makes me wonder about `promo--bg-inherit`?